app/src/main/kotlin/co/netguru/baby/monitor/client/feature/server/ChildMonitorFragment.kt

```diff
@@ -208,13 +174,23 @@ class ChildMonitorFragment : BaseDaggerFragment(), ServiceConnection,
     }
 
     private fun serverViewModelObservers() {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/b1aa35793dca6d7714e5534d6167592f0bbe6a99/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/server/ChildMonitorFragment.kt#L176"></span>The function serverViewModelObservers appears to be too complex.</td>
    </tr>
  </tbody>
</table>